runtime/near-vm-runner/src/errors.rs

```diff
@@ -80,12 +101,11 @@ impl IntoVMError for wasmer_runtime::error::RuntimeError {
                         }
                     }
                 } else {
-                    eprintln!(
+                    panic!(
```

Comment:
I think it is better for runtime to be panic-free to avoid some bug in Wasmer completely crash the network. /cc @bowenwang1996 @nearmax

Reply:
It's general problem of fail early vs. hiding the problems. I usually prefer failing early in most cases, but Max and Bowen please share your opinion.

Reply:
It is definitely better to crash the network than to continue working with diverged network or incorrect state.

Reply:
yeah it's better to explicitly panic here.